1.intro/chapter1/intro-demo/notebook.ipynb

```diff
@@ -164,23 +176,28 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "from keras.layers import Dense\n",
     "from keras.models import Sequential\n",
     "\n",
     "# Define a sequential neural network\n",
     "model = Sequential([\n",
-    "    Dense(4, input_dim=2, activation=\"relu\"),\n",
+    "    Dense(32, input_dim=2, activation=\"relu\"),\n",
     "    Dense(1, activation=\"sigmoid\")\n",
     "])\n",
     "\n",
     "# Compile the model\n",
     "model.compile(loss=\"binary_crossentropy\", optimizer='adam', metrics=['accuracy'])\n",
```

Comment:
The use of binary_crossentropy is still suspicious to me because the output layer does not represent classification probabilities.  A simpler regression loss like mean_squared_error seems more appropriate.

Reply:
It seems to perform just as well. Given that this is demo 1, I think using the simplest algorithms makes sense if they work fine. Using any of the more advanced things complicates the example more than necessary. That plus your point that it may not be ideal anyway makes a strong case to change it! I can update the PR with an adjustment here, as well. I will also try to update the commit message and PR description to explain the change.

Reply:
@csibbitt I've added a second commit to the PR, adjusting the loss function and optimization algorithm. The commit message (and PR description) include the justification based on our discussion.